variables.tf

```diff
@@ -204,6 +204,16 @@ variable "weather_api_key" {
   description = "A API key. See forecast.io"
 }
 
+variable "box_client_id" {
+  type        = "string"
+  description = "Box Client ID key. See box.com"
+}
```

Comment:
Let's add a default of an empty string to these two variables, that way existing deployments won't have to change anything unless they opt-in for box support.